internal/common/db.go

```diff
@@ -34,3 +36,41 @@ type Cluster struct {
 	// The ID of the subscription created in AMS
 	AmsSubscriptionID strfmt.UUID `json:"ams_subscription_id"`
 }
+
+func LoadHostsFromDB(db *gorm.DB, conditions ...interface{}) *gorm.DB {
+	return db.Preload("Hosts", conditions...)
+}
+
+func GetClusterFromDB(db *gorm.DB, clusterId strfmt.UUID, loadSubTables bool) (*Cluster, error) {
```

Comment:
instead of using boolean lets use a specific type, 
something like 
```
type LoadSubTablesType boolean
LoadSubTales LoadSubTableType = true
```
Then the calls will be something like
```
GetClusterFromDB(db, id, common.LoadSubTables)
GetClusterFromDB(db, id, !common.LoadSubTables)
```
I think that it's clearer then having just true/false no?

Reply:
Like